src/building/how-to-build-and-run.md

```diff
@@ -13,7 +13,9 @@ and a bunch of tools (e.g. `rustdoc`, the bootstrapping infrastructure, etc).
 
 [repo]: https://github.com/rust-lang/rust
 
-The very first step to work on `rustc` is to clone the repository:
+The very first step to work on `rustc` is to clone the repository.
+However, you might consider to fork it first.
```

Comment:
This chapter is "How to build and run the compiler" and I'm not sure how it's useful, while I totally agree if some wants to _contribute_.